exporter/awsemfexporter/metric_translator.go

```diff
@@ -376,7 +378,7 @@ func translateCWMetricToEMF(cWMetric *cWMetrics, config *Config) *cwlogs.Event {
 
 	// Create EMF metrics if there are measurements
 	// https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html#CloudWatch_Embedded_Metric_Format_Specification_structure
-	if len(cWMetric.measurements) > 0 {
+	if len(cWMetric.measurements) > 0 && !config.DisableMetricExtraction {
```

Comment:
The check here is a failsafe in case there is code in the future that adds CW measurements from a new code path and the check above doesnt suffice.